packages/idyll-cli/src/pipeline/index.js

```diff
@@ -94,6 +98,12 @@ const build = (opts, paths, resolvers) => {
         writeFile(paths.JS_OUTPUT_FILE, output.js),
         writeFile(paths.CSS_OUTPUT_FILE, output.css),
         writeFile(paths.HTML_OUTPUT_FILE, output.html),
+        pathExists(paths.STATIC_DIR).then(exists => {
+          if (exists) {
+            return copy(paths.STATIC_DIR, paths.STATIC_OUTPUT_DIR)
+          }
+          return null;
+        })
```

Comment:
Just to confirm - this gets triggered during the build, or both build and watch mode?